terms.html

```diff
@@ -3,13 +3,20 @@
 </p>
 
 <dl class="termlist">
+  <dt><dfn data-lt="author|authors|author's|author(s)|authoring">author</dfn></dt>
+  <dd>
+A role that a <a>party</a> performs as it expresses an assertion about a particular <a>entity</a>, i.e., creates a <a>claim</a>.
+  </dd>
   <dt><dfn data-lt="claims">claim</dfn></dt>
   <dd>
-An assertion made about a <a>subject</a>.
+A digital representation of an assertion made about an <a>entity</a> by a specific <a>party</a>.
+The entity is called the <a>subject</a> of the claim. 
+The <a>party</a> is referred to as the author of the claim, i.e. the one that has expressed the claim.
+The meaning of the claim (its semantics) is determined by its author.
```

Comment:
While I have no problem with this text as is, however, I think others may be misled into thinking that an author could not express this meaning through the use of a shared vocabulary that many different authors could use. This is a vitally important feature of VCs and the technology's ability to scale to a large, decentralized ecosystem. So perhaps we mention that here:

```suggestion
The meaning of the claim (its semantics) is determined by its author.
Different authors can reuse the same semantics for claims through the
use of author-independent, shared vocabularies.
```

Reply:
done.